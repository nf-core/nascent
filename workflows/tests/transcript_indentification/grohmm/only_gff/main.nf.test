nextflow_pipeline {

    name "GROHMM"
    script "../../../../../main.nf"
    tag "input"
    tag "grohmm"
    tag "gff"
    tag "chm13"
    // triggers 'bin/parameter_tuning.R', 'bin/transcriptcalling_grohmm.R'

    test("Should run groHMM with only a GFF file") {
        when {
            params {
                outdir = "$outputDir"
                skip_grohmm = false
                gff = 'https://huggingface.co/datasets/edmundmiller/nascent-test-data/resolve/main/chm13v2.0_RefSeq_Liftoff_v5.1.chr21.gff3.gz'
                // TODO Use CHM13 Fasta
                gtf = null
                bed = null
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(UTILS.removeNextflowVersion("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml")).match("software_versions") },
                { assert snapshot(
                    workflow.trace.tasks().size(),
                    path("$outputDir/transcript_identification/homer/cd4.bed"),
                    path("$outputDir/transcript_identification/homer/jurkat.bed"),
                    // FIXME Not determinstic because of the order of files
                    // Add to the other tests when fixed
                    // UTILS.getAllFilesFromDir("$outputDir/transcript_identification/pints/", ".bed"),
                    path("$outputDir/transcript_identification/intersect/").list(),
                    path("$outputDir/transcript_identification/filtered/").list(),
                    path("$outputDir/transcript_identification/intersect/").list(),
                    path("$outputDir/transcript_identification/filtered/").list(),
                    path("$outputDir/transcript_identification/grohmm/cd4.eval.txt"),
                    path("$outputDir/transcript_identification/grohmm/cd4.final.transcripts.bed"),
                    path("$outputDir/transcript_identification/grohmm/cd4.tdFinal.txt"),
                    path("$outputDir/transcript_identification/grohmm/cd4.tdplot_mqc.png").exists(),
                    path("$outputDir/transcript_identification/grohmm/cd4.transcripts.txt"),
                    path("$outputDir/transcript_identification/grohmm/jurkat.eval.txt"),
                    path("$outputDir/transcript_identification/grohmm/jurkat.final.transcripts.bed"),
                    path("$outputDir/transcript_identification/grohmm/jurkat.tdFinal.txt"),
                    path("$outputDir/transcript_identification/grohmm/jurkat.tdplot_mqc.png").exists(),
                    path("$outputDir/transcript_identification/grohmm/jurkat.transcripts.txt"),
                    // FIXME Not determinstic because of the order of files
                    // Add to the other tests when fixed
                    // path("$outputDir/quantification/").list(),
                    path("$outputDir/multiqc/multiqc_report.html").exists(),
                ).match("output_files")
                }
            )
        }
    }
}
