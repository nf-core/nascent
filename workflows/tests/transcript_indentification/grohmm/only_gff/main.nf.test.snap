{
    "Should run groHMM with only a GFF file": {
        "content": [
            112,
            {
                "BBMAP_PILEUP": {
                    "bbmap": 39.01,
                    "samtools": "1.16.1",
                    "pigz": 2.6
                },
                "BEDTOOLS_GENOMECOV_MINUS": {
                    "bedtools": "2.31.1"
                },
                "BEDTOOLS_GENOMECOV_PLUS": {
                    "bedtools": "2.31.1"
                },
                "BEDTOOLS_INTERSECT": {
                    "bedtools": "2.31.1"
                },
                "BEDTOOLS_INTERSECT_FILTER": {
                    "bedtools": "2.31.1"
                },
                "BEDTOOLS_MERGE": {
                    "bedtools": "2.31.1"
                },
                "BEDTOOLS_SORT": {
                    "bedtools": "2.31.1"
                },
                "BWA_INDEX": {
                    "bwa": "0.7.18-r1243-dirty"
                },
                "BWA_MEM": {
                    "bwa": "0.7.18-r1243-dirty",
                    "samtools": 1.2
                },
                "CAT_CAT": {
                    "pigz": "2.3.4"
                },
                "CUSTOM_GETCHROMSIZES": {
                    "getchromsizes": 1.2
                },
                "DEEPTOOLS_BAMCOVERAGE_MINUS": {
                    "deeptools": "3.5.1"
                },
                "DEEPTOOLS_BAMCOVERAGE_PLUS": {
                    "deeptools": "3.5.1"
                },
                "FASTP": {
                    "fastp": "0.23.4"
                },
                "FASTQC": {
                    "fastqc": "0.12.1"
                },
                "GFFREAD": {
                    "gffread": "0.12.7"
                },
                "GROHMM_PARAMETERTUNING": {
                    "r-base": "4.3.3",
                    "bioconductor-grohmm": "1.39.0"
                },
                "GTF2BED": {
                    "perl": "5.26.2"
                },
                "GUNZIP_FASTA": {
                    "gunzip": 1.1
                },
                "GUNZIP_GFF": {
                    "gunzip": 1.1
                },
                "HOMER_MAKETAGDIRECTORY": {
                    "homer": 4.11,
                    "samtools": 1.21
                },
                "PINTS_CALLER": {
                    "python": "3.12.6",
                    "pints": "1.1.13"
                },
                "PRESEQ_CCURVE": {
                    "preseq": "3.1.1"
                },
                "PRESEQ_LCEXTRAP": {
                    "preseq": "3.1.1"
                },
                "RSEQC_INFEREXPERIMENT": {
                    "rseqc": "5.0.2"
                },
                "RSEQC_READDISTRIBUTION": {
                    "rseqc": "5.0.2"
                },
                "RSEQC_READDUPLICATION": {
                    "rseqc": "5.0.2"
                },
                "SAMTOOLS_MERGE": {
                    "samtools": 1.21
                },
                "SUBREAD_FEATURECOUNTS_GENE": {
                    "subread": "2.0.1"
                },
                "SUBREAD_FEATURECOUNTS_PREDICTED": {
                    "subread": "2.0.1"
                },
                "Workflow": {
                    "nf-core/nascent": "v2.3.0"
                }
            },
            [
                "bwa",
                "bwa/cd4_REP1.sorted.bam",
                "bwa/cd4_REP1.sorted.bam.bai",
                "bwa/cd4_REP2.sorted.bam",
                "bwa/cd4_REP2.sorted.bam.bai",
                "bwa/jurkat.sorted.bam",
                "bwa/jurkat.sorted.bam.bai",
                "bwa/samtools_stats",
                "bwa/samtools_stats/cd4_REP1.sorted.bam.flagstat",
                "bwa/samtools_stats/cd4_REP1.sorted.bam.idxstats",
                "bwa/samtools_stats/cd4_REP1.sorted.bam.stats",
                "bwa/samtools_stats/cd4_REP2.sorted.bam.flagstat",
                "bwa/samtools_stats/cd4_REP2.sorted.bam.idxstats",
                "bwa/samtools_stats/cd4_REP2.sorted.bam.stats",
                "bwa/samtools_stats/jurkat.sorted.bam.flagstat",
                "bwa/samtools_stats/jurkat.sorted.bam.idxstats",
                "bwa/samtools_stats/jurkat.sorted.bam.stats",
                "cat",
                "cat/cd4.bed",
                "cat/jurkat.bed",
                "coverage_graphs",
                "coverage_graphs/cd4_REP1.dreg.bedGraph",
                "coverage_graphs/cd4_REP1.minus.bedGraph",
                "coverage_graphs/cd4_REP1.minus.bigWig",
                "coverage_graphs/cd4_REP1.plus.bedGraph",
                "coverage_graphs/cd4_REP1.plus.bigWig",
                "coverage_graphs/cd4_REP2.dreg.bedGraph",
                "coverage_graphs/cd4_REP2.minus.bedGraph",
                "coverage_graphs/cd4_REP2.minus.bigWig",
                "coverage_graphs/cd4_REP2.plus.bedGraph",
                "coverage_graphs/cd4_REP2.plus.bigWig",
                "coverage_graphs/jurkat.dreg.bedGraph",
                "coverage_graphs/jurkat.minus.bedGraph",
                "coverage_graphs/jurkat.minus.bigWig",
                "coverage_graphs/jurkat.plus.bedGraph",
                "coverage_graphs/jurkat.plus.bigWig",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/fastqc-status-check-heatmap.txt",
                "multiqc/multiqc_data/fastqc_adapter_content_plot.txt",
                "multiqc/multiqc_data/fastqc_overrepresented_sequences_plot.txt",
                "multiqc/multiqc_data/fastqc_per_base_n_content_plot.txt",
                "multiqc/multiqc_data/fastqc_per_base_sequence_quality_plot.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_quality_scores_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_counts_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_duplication_levels_plot.txt",
                "multiqc/multiqc_data/fastqc_top_overrepresented_sequences_table.txt",
                "multiqc/multiqc_data/homer-tag-directory-gc-content.txt",
                "multiqc/multiqc_data/homer-tag-info-dist.txt",
                "multiqc/multiqc_data/homer-tag-length-dist.txt",
                "multiqc/multiqc_data/homer_tagdir.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_fastqc.txt",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_rseqc_infer_experiment.txt",
                "multiqc/multiqc_data/multiqc_rseqc_read_distribution.txt",
                "multiqc/multiqc_data/multiqc_samtools_flagstat.txt",
                "multiqc/multiqc_data/multiqc_samtools_idxstats.txt",
                "multiqc/multiqc_data/multiqc_samtools_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_data/preseq.txt",
                "multiqc/multiqc_data/preseq_complexity_plot_molecules.txt",
                "multiqc/multiqc_data/rseqc_infer_experiment_plot.txt",
                "multiqc/multiqc_data/rseqc_read_distribution_plot.txt",
                "multiqc/multiqc_data/rseqc_read_dups.txt",
                "multiqc/multiqc_data/rseqc_read_dups_plot.txt",
                "multiqc/multiqc_data/samtools-flagstat-dp_Percentage_of_total.txt",
                "multiqc/multiqc_data/samtools-flagstat-dp_Read_counts.txt",
                "multiqc/multiqc_data/samtools-idxstats-mapped-reads-plot_Normalised_Counts.txt",
                "multiqc/multiqc_data/samtools-idxstats-mapped-reads-plot_Observed_over_Expected_Counts.txt",
                "multiqc/multiqc_data/samtools-idxstats-mapped-reads-plot_Raw_Counts.txt",
                "multiqc/multiqc_data/samtools-stats-dp.txt",
                "multiqc/multiqc_data/samtools_alignment_plot.txt",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_nascent_software_mqc_versions.yml",
                "preprocessing",
                "preprocessing/fastp",
                "preprocessing/fastp/cd4_REP1.trimmed.fastp.html",
                "preprocessing/fastp/cd4_REP1.trimmed.fastp.json",
                "preprocessing/fastp/cd4_REP1.trimmed.fastp.log",
                "preprocessing/fastp/cd4_REP2.trimmed.fastp.html",
                "preprocessing/fastp/cd4_REP2.trimmed.fastp.json",
                "preprocessing/fastp/cd4_REP2.trimmed.fastp.log",
                "preprocessing/fastp/jurkat.trimmed.fastp.html",
                "preprocessing/fastp/jurkat.trimmed.fastp.json",
                "preprocessing/fastp/jurkat.trimmed.fastp.log",
                "preprocessing/fastqc",
                "preprocessing/fastqc/cd4_REP1_fastqc.html",
                "preprocessing/fastqc/cd4_REP1_fastqc.zip",
                "preprocessing/fastqc/cd4_REP2_fastqc.html",
                "preprocessing/fastqc/cd4_REP2_fastqc.zip",
                "preprocessing/fastqc/jurkat_fastqc.html",
                "preprocessing/fastqc/jurkat_fastqc.zip",
                "quality_control",
                "quality_control/bbsplit",
                "quality_control/bbsplit/cd4_REP1.coverage.hist.txt",
                "quality_control/bbsplit/cd4_REP1.coverage.stats.txt",
                "quality_control/bbsplit/cd4_REP2.coverage.hist.txt",
                "quality_control/bbsplit/cd4_REP2.coverage.stats.txt",
                "quality_control/bbsplit/jurkat.coverage.hist.txt",
                "quality_control/bbsplit/jurkat.coverage.stats.txt",
                "quality_control/preseq",
                "quality_control/preseq/cd4_REP1.c_curve.txt",
                "quality_control/preseq/cd4_REP1.lc_extrap.txt",
                "quality_control/preseq/cd4_REP2.c_curve.txt",
                "quality_control/preseq/cd4_REP2.lc_extrap.txt",
                "quality_control/preseq/jurkat.c_curve.txt",
                "quality_control/preseq/jurkat.lc_extrap.txt",
                "quality_control/preseq/log",
                "quality_control/preseq/log/cd4_REP1.command.log",
                "quality_control/preseq/log/cd4_REP2.command.log",
                "quality_control/preseq/log/jurkat.command.log",
                "quality_control/rseqc",
                "quality_control/rseqc/infer_experiment",
                "quality_control/rseqc/infer_experiment/cd4_REP1.infer_experiment.txt",
                "quality_control/rseqc/infer_experiment/cd4_REP2.infer_experiment.txt",
                "quality_control/rseqc/infer_experiment/jurkat.infer_experiment.txt",
                "quality_control/rseqc/read_distribution",
                "quality_control/rseqc/read_distribution/cd4_REP1.read_distribution.txt",
                "quality_control/rseqc/read_distribution/cd4_REP2.read_distribution.txt",
                "quality_control/rseqc/read_distribution/jurkat.read_distribution.txt",
                "quality_control/rseqc/read_duplication",
                "quality_control/rseqc/read_duplication/pdf",
                "quality_control/rseqc/read_duplication/pdf/cd4_REP1.DupRate_plot.pdf",
                "quality_control/rseqc/read_duplication/pdf/cd4_REP2.DupRate_plot.pdf",
                "quality_control/rseqc/read_duplication/pdf/jurkat.DupRate_plot.pdf",
                "quality_control/rseqc/read_duplication/rscript",
                "quality_control/rseqc/read_duplication/rscript/cd4_REP1.DupRate_plot.r",
                "quality_control/rseqc/read_duplication/rscript/cd4_REP2.DupRate_plot.r",
                "quality_control/rseqc/read_duplication/rscript/jurkat.DupRate_plot.r",
                "quality_control/rseqc/read_duplication/xls",
                "quality_control/rseqc/read_duplication/xls/cd4_REP1.pos.DupRate.xls",
                "quality_control/rseqc/read_duplication/xls/cd4_REP1.seq.DupRate.xls",
                "quality_control/rseqc/read_duplication/xls/cd4_REP2.pos.DupRate.xls",
                "quality_control/rseqc/read_duplication/xls/cd4_REP2.seq.DupRate.xls",
                "quality_control/rseqc/read_duplication/xls/jurkat.pos.DupRate.xls",
                "quality_control/rseqc/read_duplication/xls/jurkat.seq.DupRate.xls",
                "quantification",
                "quantification/gene",
                "quantification/gene/cd4.featureCounts.txt",
                "quantification/gene/cd4.featureCounts.txt.summary",
                "quantification/gene/jurkat.featureCounts.txt",
                "quantification/gene/jurkat.featureCounts.txt.summary",
                "quantification/nascent",
                "quantification/nascent/cd4-group_cd4_intersect-transcripts.featureCounts.txt",
                "quantification/nascent/cd4-group_cd4_intersect-transcripts.featureCounts.txt.summary",
                "quantification/nascent/cd4-group_jurkat_intersect-transcripts.featureCounts.txt",
                "quantification/nascent/cd4-group_jurkat_intersect-transcripts.featureCounts.txt.summary",
                "quantification/nascent/jurkat-group_cd4_intersect-transcripts.featureCounts.txt",
                "quantification/nascent/jurkat-group_cd4_intersect-transcripts.featureCounts.txt.summary",
                "quantification/nascent/jurkat-group_jurkat_intersect-transcripts.featureCounts.txt",
                "quantification/nascent/jurkat-group_jurkat_intersect-transcripts.featureCounts.txt.summary",
                "samtools",
                "samtools/cd4.bam",
                "samtools/jurkat.bam",
                "transcript_identification",
                "transcript_identification/filtered",
                "transcript_identification/filtered/cd4_filtered.bed",
                "transcript_identification/filtered/jurkat_filtered.bed",
                "transcript_identification/grohmm",
                "transcript_identification/grohmm/cd4.SE.tuning.csv",
                "transcript_identification/grohmm/cd4.eval.txt",
                "transcript_identification/grohmm/cd4.final.transcripts.bed",
                "transcript_identification/grohmm/cd4.tdFinal.txt",
                "transcript_identification/grohmm/cd4.tdFinal_mqc.csv",
                "transcript_identification/grohmm/cd4.tdplot_mqc.png",
                "transcript_identification/grohmm/cd4.transcripts.txt",
                "transcript_identification/grohmm/jurkat.SE.tuning.csv",
                "transcript_identification/grohmm/jurkat.eval.txt",
                "transcript_identification/grohmm/jurkat.final.transcripts.bed",
                "transcript_identification/grohmm/jurkat.tdFinal.txt",
                "transcript_identification/grohmm/jurkat.tdFinal_mqc.csv",
                "transcript_identification/grohmm/jurkat.tdplot_mqc.png",
                "transcript_identification/grohmm/jurkat.transcripts.txt",
                "transcript_identification/homer",
                "transcript_identification/homer/cd4.bed",
                "transcript_identification/homer/cd4.bedGraph.gz",
                "transcript_identification/homer/cd4.peaks.txt",
                "transcript_identification/homer/cd4_tagdir",
                "transcript_identification/homer/cd4_tagdir/chr21.tags.tsv",
                "transcript_identification/homer/cd4_tagdir/genomeGCcontent.txt",
                "transcript_identification/homer/cd4_tagdir/tagAutocorrelation.txt",
                "transcript_identification/homer/cd4_tagdir/tagCountDistribution.txt",
                "transcript_identification/homer/cd4_tagdir/tagFreq.txt",
                "transcript_identification/homer/cd4_tagdir/tagFreqUniq.txt",
                "transcript_identification/homer/cd4_tagdir/tagGCcontent.txt",
                "transcript_identification/homer/cd4_tagdir/tagInfo.txt",
                "transcript_identification/homer/cd4_tagdir/tagLengthDistribution.txt",
                "transcript_identification/homer/jurkat.bed",
                "transcript_identification/homer/jurkat.bedGraph.gz",
                "transcript_identification/homer/jurkat.peaks.txt",
                "transcript_identification/homer/jurkat_tagdir",
                "transcript_identification/homer/jurkat_tagdir/chr21.tags.tsv",
                "transcript_identification/homer/jurkat_tagdir/genomeGCcontent.txt",
                "transcript_identification/homer/jurkat_tagdir/tagAutocorrelation.txt",
                "transcript_identification/homer/jurkat_tagdir/tagCountDistribution.txt",
                "transcript_identification/homer/jurkat_tagdir/tagFreq.txt",
                "transcript_identification/homer/jurkat_tagdir/tagFreqUniq.txt",
                "transcript_identification/homer/jurkat_tagdir/tagGCcontent.txt",
                "transcript_identification/homer/jurkat_tagdir/tagInfo.txt",
                "transcript_identification/homer/jurkat_tagdir/tagLengthDistribution.txt",
                "transcript_identification/homer/versions.yml",
                "transcript_identification/intersect",
                "transcript_identification/intersect/cd4_intersect.bed",
                "transcript_identification/intersect/jurkat_intersect.bed",
                "transcript_identification/merged",
                "transcript_identification/merged/cd4_merged.bed",
                "transcript_identification/merged/jurkat_merged.bed",
                "transcript_identification/pints",
                "transcript_identification/pints/cd4_chr21_1_unidirectional_peaks.bed",
                "transcript_identification/pints/jurkat_chr21_1_unidirectional_peaks.bed"
            ],
            [
                "cd4_REP1.sorted.bam.flagstat:md5,98ee509d07c1db03ccce634ccf410d07",
                "cd4_REP1.sorted.bam.idxstats:md5,3f30696c4c8628b47029f63d9521f0e7",
                "cd4_REP1.sorted.bam.stats:md5,4902e9a7e501f6150549e195b7dc44c0",
                "cd4_REP2.sorted.bam.flagstat:md5,235f511666cc191d1ba0d0e7334eb6ad",
                "cd4_REP2.sorted.bam.idxstats:md5,5eb0e894286a602d8c34b18ae226863e",
                "cd4_REP2.sorted.bam.stats:md5,77abd3727f54071ce24ac389a91934ed",
                "jurkat.sorted.bam.flagstat:md5,a1eec107305fe681a02771574de5c5fb",
                "jurkat.sorted.bam.idxstats:md5,02590cdcaec907c42cf6507d62844a4c",
                "jurkat.sorted.bam.stats:md5,824ca1525abba55d54e9df2c319be932",
                "cd4.bed:md5,60e0da7e5691e55d86eb9df9f6ea0c46",
                "jurkat.bed:md5,3584ff1a08cdecc92b6fcf6b2db8dc90",
                "cd4_REP1.dreg.bedGraph:md5,35eb3dce09cf8a32e71fe459e0f4a0a5",
                "cd4_REP1.minus.bedGraph:md5,141abb85ef58f8cd77b3908ac22a35c6",
                "cd4_REP1.minus.bigWig:md5,52d93314a403c1303e91c34ea731b445",
                "cd4_REP1.plus.bedGraph:md5,fac7dc18180431b03e367da72383da88",
                "cd4_REP1.plus.bigWig:md5,8b869ed53753eafa00dcc63070078883",
                "cd4_REP2.dreg.bedGraph:md5,b1fc188a70aa795a38649954fc41896c",
                "cd4_REP2.minus.bedGraph:md5,5ff711ae9e757b1a60364131b8003880",
                "cd4_REP2.minus.bigWig:md5,deb81acab23d25a991e26af82b33b04e",
                "cd4_REP2.plus.bedGraph:md5,c48f7894fabd484661226fde193a67fb",
                "cd4_REP2.plus.bigWig:md5,f308509ebc485ed7b1ba8616aba8e400",
                "jurkat.dreg.bedGraph:md5,f5c6f0e6035c2d3f1cb803f532c3fb93",
                "jurkat.minus.bedGraph:md5,4da08f7094f8ff6c2e1ab33ea982c328",
                "jurkat.minus.bigWig:md5,378935c83fe428a878cb91490ce5e297",
                "jurkat.plus.bedGraph:md5,1e801978023f381e151403b572a9bd70",
                "jurkat.plus.bigWig:md5,59bbb8db066fb91c82685eff1c675823",
                "cd4_REP1.trimmed.fastp.json:md5,bd9a3344c1591d6be4d524451f9dca53",
                "cd4_REP2.trimmed.fastp.json:md5,bc6e3f9ff7835f220535cc393b8eb25f",
                "jurkat.trimmed.fastp.json:md5,65f7247a87479c7157c7357d575336e2",
                "cd4.SE.tuning.csv:md5,9d76e9a83079d587459fda16af8c4da8",
                "cd4.eval.txt:md5,de1f790f721a67dfd1e1bf1592a7971a",
                "cd4.final.transcripts.bed:md5,3707df9ae8e7a95b6b5bfc336cfb1c8a",
                "cd4.tdFinal.txt:md5,f8db76ddb3212459405e0d780c1dc807",
                "cd4.tdFinal_mqc.csv:md5,a227d42afbcc590b4e949075cde4a5b6",
                "cd4.tdplot_mqc.png:md5,159224ca8b99bdf67f3f7d328bd741dd",
                "cd4.transcripts.txt:md5,48d15103e5f67abc30b4da71a595243b",
                "jurkat.SE.tuning.csv:md5,d339a8072a8789896d4df6b78eeef6a2",
                "jurkat.eval.txt:md5,f8f6402c871cd55d6cfb9c2be04627b6",
                "jurkat.final.transcripts.bed:md5,cd3ce78611dcad5a1893a6da3d70a39f",
                "jurkat.tdFinal.txt:md5,4bb6a220a6ba29941d4ab28696816afd",
                "jurkat.tdFinal_mqc.csv:md5,a227d42afbcc590b4e949075cde4a5b6",
                "jurkat.tdplot_mqc.png:md5,91825f59df971e9e79256058da010f3f",
                "jurkat.transcripts.txt:md5,901825bc4b6b93e718f7082fb4c2c579",
                "cd4.bed:md5,202652821e59d0357c699b217d02f755",
                "cd4.peaks.txt:md5,84b08e639d17d480cbc90b465ecfff73",
                "jurkat.bed:md5,7c37a46578dac48f8bf15155cd78b214",
                "jurkat.peaks.txt:md5,4cc5640d1e7c84db1116d6514265a4c3",
                "versions.yml:md5,7c0dcd0a18b3c753def73b96cb825792",
                "cd4_merged.bed:md5,f79432639c3f1b86787947e8d7242c75",
                "jurkat_merged.bed:md5,a94e1ec47f0564122ae1eb0f73be4ebd",
                "cd4_chr21_1_unidirectional_peaks.bed:md5,60e0da7e5691e55d86eb9df9f6ea0c46",
                "jurkat_chr21_1_unidirectional_peaks.bed:md5,3584ff1a08cdecc92b6fcf6b2db8dc90"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.2"
        },
        "timestamp": "2024-12-18T08:07:15.190593"
    }
}
